Automattic/jetpack
# Components: Update the Jetpack Social icon

#### Changes proposed in this Pull Request:
Now we have the final design for the icon, this PR updates the SVG to
the optimised verison.

#### Jetpack product discussion
p1HpG7-g1B-p2

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Run storybook `cd projects/js-packages/storybook && pnpm run storybook:dev`
* Head to Components -> Icons and check that the Social icon looks ok.

<img width="279" alt="image" src="https://user-images.githubusercontent.com/96462/170135495-fe4e1aea-c48c-45a0-8fd9-4dae44a255e7.png">
